clients/cli/src/main.rs

```diff
@@ -169,34 +254,24 @@ fn parse_and_run() -> Result<(), CliError> {
 
     let core = Core::init(&Config { logs: true, writeable_path })?;
 
+    use crate::Lockbook::*;
```

Comment:
I don't think `crate` necessary for this one.

Reply:
2a1ba9bb